tycho-its/projects/product.programArgs/product.product

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?pde version="3.5"?>
+
+<product uid="ppa.product" id="product" application="application" version="1.0.0.qualifier" useFeatures="false" includeLaunchers="false">
+
+   <launcherArgs>
+      <programArgs>-blah1 -blah2 -configuration @user.dir/configuration
```

Comment:
have you tried writing each arg in one line?

Reply:
Yes, still fails. If you want I could add another product with multiline args